src/main/java/at/hannibal2/skyhanni/config/features/gui/GUIConfig.java

```diff
@@ -31,6 +32,10 @@ public class GUIConfig {
     @Category(name = "Custom Scoreboard", desc = "Custom Scoreboard Settings")
     public CustomScoreboardConfig customScoreboard = new CustomScoreboardConfig();
 
+    @Expose
+    @Category(name = "Enchant Parsing", desc = "Settings for Skyhanni's Enchant Parsing")
+    public EnchantParsingConfig enchantParsing = new EnchantParsingConfig();
```

Comment:
move to inventory. (this is not a gui, this is a change to item, what is part of invenotry in the config strucutre)